source/particle/property/crystal_preferred_orientation.cc

```diff
@@ -711,16 +711,16 @@ namespace aspect
                   }
                 beta[indices.back()] = 0.0;
 
-                // Now compute the crystal rate of deformation tensor.
-                for (unsigned int i = 0; i < 3; ++i)
+                // Now compute the crystal rate of deformation tensor. equation 4 of Kaminski&Ribe 2001
+                // rotation_matrix_transposed = inverse of rotation matrix
+                // (see Engler et al., 2024 book: Intro to Texture analysis chp 2.3.2 the rotAtion MAtrix)
+                // this transform the crystal reference frame to specimen reference frame
+                for (unsigned int slip_system_i = 0; slip_system_i < 4; ++slip_system_i)
```

Comment:
The last slip system is not computed because it is not active. At least this was the case in D-Rex (fortran, matlab and aspect). Is there are reason you now want to compute this?

Reply:
I am not sure why more than four slip systems is not included in the original DRex. I am guess it is due to its focus was olivine and so only four is needed.
However, Bascou et al., 2002, even though it was using VPSC not DRex, it had 12 slip systems.   Also, ECOMAN the newest code based on DRex and  from the same group had 12 slip systems. So I am going to try out and research about if there is such limit in the max number of slip systems within DRex algorithm.

Bascou, J., Tommasi, A., & Mainprice, D. (2002). Plastic deformation and development of clinopyroxene lattice preferred orientations in eclogites. Journal of Structural Geology, 24(8), 1357–1368.

Faccenda, M., VanderBeek, B. P., de Montserrat, A., Yang, J., & Ribe, N. (2024). ECOMAN: an open-source package for geodynamic and seismological modeling of mechanical anisotropy. EGUsphere, 2024, 1–27.